modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -635,6 +645,133 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
         });
     }
 
+    private CompletableFuture<Boolean> initTableFsmOnTableCreate(CreateTableEventParameters parameters) {
```

Comment:
The most confusing method in the PR, to my opinion.

Reply:
The name or in general? Let's discuss the name here https://github.com/apache/ignite-3/pull/3931#discussion_r1687205483

Reply:
Nope, not the name, but the method body.